examples/spl_composites/com.ibm.streamsx.testing.python/testCustom.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from streamsx.topology.topology import *
+from streamsx.topology.tester import Tester
+import streamsx.spl.op as op
+import streamsx.spl.toolkit as tk
+import numpy as np
+
+# TestCustomStandalone class for driving the test in standalone mode
+class TestCustomStandalone(unittest.TestCase):
```

Comment:
should lead by example on naming, this is testing the MultipyBy operator, not 'Custom', so rename to `TestMultipyByStandalone`.

Reply:
done